tests/phpunit/data/themedir1/block-theme/styles/variation-b.json

```diff
@@ -14,5 +14,12 @@
 				}
 			}
 		}
+	},
+	"styles": {
+		"background": {
+			"backgroundImage": {
+				"url": "file:./assets/sugarloaf-mountain.jpg"
```

Comment:
Checking if adding the above image is okay, e.g., no side effects for SVN or anything else I'm ignorant of 😄 

Actually, now I'm thinking about it... it's probably useless since we're overloading the filter in the tests anyway.

Reply:
Deleted.